react-app/hooks/useFileDownloader.js

```diff
@@ -16,13 +16,17 @@ const useFileDownloader = ({ onError, onFileDownloaded}) => {
   const [, setBlobUrl] = useState(null);
   const [isLoading, setIsLoading] = useState(false);
 
-  const downloadFileHook = async (fileUrl, fileName, isZip) => {
+  const downloadFileHook = async (fileUrl, fileName, isZip, listOfFiles) => {
 
     setIsLoading(true);
 
     try {
       saveAs(fileUrl, fileName);
-      onFileDownloaded();
+      try {
+        onFileDownloaded(listOfFiles);
+      }catch (error) {
+        console.debug("FileManager Xblock: We could not log the download");
+      }
```

Comment:
is this try-catch block still needed?

Reply:
Ideally now it should not break, but even if it breaks it should not show the UI error that `onError` in L31 presents to the user, so I think it is better to leave it.